### PR TITLE
logging-collector-logs-writer clusterrole is not required to be created manually

### PR DESCRIPTION
**Pull request by prithvipatil97**
- Inside `About Logging 6.0` [1]  section in `Quick Start` [2] , the step number: 4, Create a ClusterRole for the collector: is not required.

[1] https://docs.openshift.com/container-platform/4.16/observability/logging/logging-6.0/log6x-about.html
[2] https://docs.openshift.com/container-platform/4.16/observability/logging/logging-6.0/log6x-about.html#quick-start

- In step 4, it has been mentioned that the user needs to create the ClusterRole `logging-collector-logs-writer` 
- But this `ClusterRole` is created automatically by the Loki operator and it is not required to create it manually.

- This ClusterRole was also made available with the `Loki Operator 5.9` version.
- Here is jira [3] for reference: 

[3] https://issues.redhat.com/browse/LOG-5967 

- Hence step 4 needs to be omitted from documentation:

------
4. Create a ClusterRole for the collector: 
~~~
apiVersion: rbac.authorization.k8s.io/v1
kind: ClusterRole
metadata:
  name: logging-collector-logs-writer
rules:
- apiGroups:
  - loki.grafana.com
  resourceNames:
  - logs
  resources:
  - application
  - audit
  - infrastructure
  verbs:
  - create
~~~
------

Version(s):

RHOCP-4.18, RHOCP-4.17, RHOCP-4.16, RHOCP-4.15, RHOCP-4.14

Issue:

https://issues.redhat.com/browse/OBSDOCS-1341

Link to docs preview:

https://86531--ocpdocs-pr.netlify.app/openshift-enterprise/latest/observability/logging/logging-6.0/log6x-about.html

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Review by skopacz1 — APPROVED** [review]
Peer review LGTM! However, a few friendly reminders before proceeding to merge review:

- Please make sure to get QE review for this PR, and once you have gotten a review, check off the box in the PR description
- 4.18 is not out yet, but if this change should apply to versions 4.15 and later, please also mention 4.18 in the PR description (you can just write 4.15+ if you'd like)

**Comment by prithvipatil97** [comment]
Hello @skopacz1 ,
Thank you for reviewing this PR and providing the approval.

Yes, definitely. I will take a QE approval before submitting this PR for merge.
Regarding RHOCP versions, this changes needs to be applied from 4.14.
Sure I will add RHOCP-4.18 in the PR description.

Thanks again.

Regards,
Prithviraj Patil

**Comment by prithvipatil97** [comment]
Hello Team,
I need QE approval for this change.

All tests have passed and Peer review is also done.

@anpingli  , @kabirbhartiRH  , @QiaolingTang  , It would be really helpful if someone could please take a look and provide QE approval for this change.

Thanks in advance.

Regards,
Prithviraj Patil

**Comment by QiaolingTang** [comment]
LGTM.

**Comment by prithvipatil97** [comment]
Hello @QiaolingTang ,
Thank you very much for providing the QE approval for this change.

Regards,
Prithviraj Patil

**Review by xenolinux — COMMENTED** [review]
Merge review LGTM

**Pull request merged**
